### PR TITLE
Dịch "alexnet_vn" - Phần 8

### LINKED ISSUE #1330: Dịch "alexnet_vn" - Phần 8
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_convolutional-modern/alexnet_vn.md#L425-L482

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by nguyenduydu**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #1330

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [x] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_convolutional-modern/alexnet_vn.md [hunk 1]
```diff
@@ -456,7 +457,10 @@ d2l.train_ch6(net, train_iter, test_iter, num_epochs, lr)
 * Dropout, ReLU and preprocessing were the other key steps in achieving excellent performance in computer vision tasks.
 -->
 
-*dịch đoạn phía trên*
+* AlexNet có cấu trúc tương tự như LeNet, nhưng sử dụng nhiều tầng tích chập hơn và không gian tham số lớn hơn để khớp tập dữ liệu ImageNet có kích thước lớn.
+* Ngày nay AlexNet đã bị vượt qua bởi các kiến trúc hiệu quả hơn nhiều nhưng nó là một bước quan trọng từ các mạng nông đến các mạng sâu được sử dụng ngày nay.
+* Mặc dù có vẻ như chỉ có thêm một vài dòng trong cách lập trình của AlexNet so với LeNet, nhưng cộng đồng học thuật đã phải mất nhiều năm để đón nhận sự thay đổi khái niệm này và tận dụng kết quả thực nghiệm tuyệt vời của nó. Điều này cũng là do việc thiếu các công cụ tính toán hiệu quả.
```
Comment (rootonchair) [inline chapter_convolutional-modern/alexnet_vn.md:462]:
```suggestion
* Mặc dù có vẻ như chỉ tốn thêm một vài dòng trong mã nguồn của AlexNet so với LeNet, nhưng cộng đồng học thuật đã phải mất nhiều năm để đón nhận sự thay đổi khái niệm này và tận dụng kết quả thực nghiệm tuyệt vời của nó. Một phần cũng là do sự thiếu thốn của các công cụ tính toán hiệu quả.
```

File: chapter_convolutional-modern/alexnet_vn.md [hunk 2]
```diff
@@ -456,7 +457,10 @@ d2l.train_ch6(net, train_iter, test_iter, num_epochs, lr)
...
+* Dropout, ReLU và tiền xử lý là những bước quan trọng khác để đạt được hiệu suất tuyệt vời trong các bài toán thị giác máy tính.
```
Comment (minhduc0711) [inline chapter_convolutional-modern/alexnet_vn.md:463]:
```suggestion
* Dropout, ReLU và tiền xử lý là những bước quan trọng khác để đạt được kết quả rất tốt trong các bài toán thị giác máy tính.
```

File: chapter_convolutional-modern/alexnet_vn.md [hunk 3]
```diff
@@ -456,7 +457,10 @@ d2l.train_ch6(net, train_iter, test_iter, num_epochs, lr)
 * Dropout, ReLU and preprocessing were the other key steps in achieving excellent performance in computer vision tasks.
 -->
 
-*dịch đoạn phía trên*
+* AlexNet có cấu trúc tương tự như LeNet, nhưng sử dụng nhiều tầng tích chập hơn và không gian tham số lớn hơn để khớp tập dữ liệu ImageNet có kích thước lớn.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/alexnet_vn.md:460]:
```suggestion
* AlexNet có cấu trúc tương tự như LeNet, nhưng sử dụng nhiều tầng tích chập hơn với không gian tham số lớn hơn để khớp tập dữ liệu ImageNet có kích thước lớn.
```

File: chapter_convolutional-modern/alexnet_vn.md (hunk 1 above)
Comment (lkhphuc) [inline chapter_convolutional-modern/alexnet_vn.md:462]:
```suggestion
* Mặc dù có vẻ như chỉ có thêm một vài dòng trong cách lập trình của AlexNet so với LeNet, nhưng cộng đồng học thuật đã phải mất nhiều năm để đón nhận quan điểm đổi mới này và tận dụng kết quả thực nghiệm tuyệt vời của nó. Một phần nữa cũng là do việc thiếu các công cụ tính toán hiệu quả.
```

File: chapter_convolutional-modern/alexnet_vn.md (hunk 2 above)
Comment (lkhphuc) [inline chapter_convolutional-modern/alexnet_vn.md:463]:
```suggestion
* Dropout, ReLU và tiền xử lý là những bước quan trọng khác để đạt được hiệu suất tuyệt vời trong các tác vụ thị giác máy tính.
```

**Comment by nguyenduydu** [comment]
Cám ơn mọi người đã góp ý.

**Comment by minhduc0711** [comment]
@nguyenduydu còn phần bài tập nữa chưa dịch đó

**Comment by nguyenduydu** [comment]
@minhduc0711  hôm qua mình thấy comment nhắc rồi mà quên không hiểu sao bỏ qua. Thanks for reminding me :))

**Comment by nguyenduydu** [comment]
@minhduc0711  mình đã thêm phần dịch cho phần bài tập, bạn review giúp mình nhé. Đặc biệt có bài số 4 mình chưa tìm được cách hài lòng nhất, mình đã comment lại. Thanks.

File: chapter_convolutional-modern/alexnet_vn.md [hunk 4]
```diff
@@ -477,7 +481,16 @@ d2l.train_ch6(net, train_iter, test_iter, num_epochs, lr)
...
+1. Thử tăng số lượng epoch và xem kết quả khác nhau như thế nào so với LeNet? Tại sao lại có sự khác nhau như vậy?
+2. AlexNet có thể là quá phức tạp đối với tập dữ liệu Fashion-MNIST.
+     * Thử đơn giản hóa mô hình để làm cho việc huấn luyện trở nên nhanh hơn nhưng đồng thời vẫn đảm bảo rằng độ chính xác không bị giảm đi đáng kể.
+     * Thử thiết kế một mô hình tốt hơn mà có thể hoạt động trực tiếp trên các ảnh có kích thước $28 \times 28$.
+3. Điều chỉnh kích thước batch và quan sát các thay đổi về độ chính xác và việc tiêu thụ bộ nhớ GPU.
+4. Rooflines: <!-- mình chưa tìm được cách dịch cho từ rooflines -->
+     * Phần nào trong thiết kế của AlexNet có ảnh hưởng lớn tới việc sử dụng bộ nhớ của mạng này?
+     * Phần nào trong thiết kế của AlexNet có ảnh hưởng lớn tới việc tính toán của mạng này?
+     * Phần nào trong thiết kế của AlexNet có ảnh hưởng lớn tới băng thông bộ nhớ khi tính toán kết quả?
+5. Áp dụng dropout và ReLU cho LeNet5. Độ chính xác của mô hình có được cải thiện hay không? Dữ liệu được tiền xử lý như thế nào?
```
Comment (duythanhvn) [inline chapter_convolutional-modern/alexnet_vn.md:493]:
```suggestion
5. Khi áp dụng dropout và ReLU cho LeNet5, độ chính xác của mô hình có được cải thiện hay không? Dữ liệu được tiền xử lý như thế nào?
```

File: chapter_convolutional-modern/alexnet_vn.md [hunk 5]
```diff
@@ -477,7 +481,16 @@ d2l.train_ch6(net, train_iter, test_iter, num_epochs, lr)
 5. Apply dropout and ReLU to LeNet5. Does it improve? How about preprocessing?
 -->
 
-*dịch đoạn phía trên*
+1. Thử tăng số lượng epoch và xem kết quả khác nhau như thế nào so với LeNet? Tại sao lại có sự khác nhau như vậy?
+2. AlexNet có thể là quá phức tạp đối với tập dữ liệu Fashion-MNIST.
```
Comment (duythanhvn) [inline chapter_convolutional-modern/alexnet_vn.md:485]:
```suggestion
2. AlexNet có thể là quá phức tạp đối với tập dữ liệu Fashion-MNIST. Vậy: 
```

File: chapter_convolutional-modern/alexnet_vn.md [hunk 6]
```diff
@@ -477,7 +481,16 @@ d2l.train_ch6(net, train_iter, test_iter, num_epochs, lr)
...
+     * Thử đơn giản hóa mô hình để làm cho việc huấn luyện trở nên nhanh hơn nhưng đồng thời vẫn đảm bảo rằng độ chính xác không bị giảm đi đáng kể.
```
Comment (duythanhvn) [inline chapter_convolutional-modern/alexnet_vn.md:486]:
```suggestion
     * Hãy thử đơn giản hóa mô hình để làm cho việc huấn luyện trở nên nhanh hơn nhưng đồng thời vẫn đảm bảo rằng độ chính xác không bị giảm đi đáng kể.
```

File: chapter_convolutional-modern/alexnet_vn.md [hunk 7]
```diff
@@ -477,7 +481,16 @@ d2l.train_ch6(net, train_iter, test_iter, num_epochs, lr)
...
+     * Thử thiết kế một mô hình tốt hơn mà có thể hoạt động trực tiếp trên các ảnh có kích thước $28 \times 28$.
```
Comment (duythanhvn) [inline chapter_convolutional-modern/alexnet_vn.md:487]:
```suggestion
     * Hãy thử thiết kế một mô hình tốt hơn có thể hoạt động trực tiếp trên các ảnh có kích thước $28 \times 28$.
```

File: chapter_convolutional-modern/alexnet_vn.md [hunk 8]
```diff
@@ -477,7 +481,16 @@ d2l.train_ch6(net, train_iter, test_iter, num_epochs, lr)
...
+3. Điều chỉnh kích thước batch và quan sát các thay đổi về độ chính xác và việc tiêu thụ bộ nhớ GPU.
+4. Rooflines: <!-- mình chưa tìm được cách dịch cho từ rooflines -->
+     * Phần nào trong thiết kế của AlexNet có ảnh hưởng lớn tới việc sử dụng bộ nhớ của mạng này?
+     * Phần nào trong thiết kế của AlexNet có ảnh hưởng lớn tới việc tính toán của mạng này?
+     * Phần nào trong thiết kế của AlexNet có ảnh hưởng lớn tới băng thông bộ nhớ khi tính toán kết quả?
```
Comment (duythanhvn) [inline chapter_convolutional-modern/alexnet_vn.md:492]:
```suggestion
4. Phần nào trong thiết kế của AlexNet có ảnh hưởng lớn đến:
     * Việc sử dụng bộ nhớ của mạng này?
     * Sự tính toán của mạng này?
     * Băng thông bộ nhớ khi tính toán kết quả?
```

File: chapter_convolutional-modern/alexnet_vn.md [hunk 9]
```diff
@@ -477,7 +481,16 @@ d2l.train_ch6(net, train_iter, test_iter, num_epochs, lr)
 5. Apply dropout and ReLU to LeNet5. Does it improve? How about preprocessing?
 -->
 
-*dịch đoạn phía trên*
+1. Thử tăng số lượng epoch và xem kết quả khác nhau như thế nào so với LeNet? Tại sao lại có sự khác nhau như vậy?
```
Comment (lkhphuc) [inline chapter_convolutional-modern/alexnet_vn.md:484]:
```suggestion
1. Thử tăng số lượng epoch và xem kết quả khác như thế nào so với LeNet? Tại sao lại có sự khác nhau như vậy?
```

**Comment by nguyenduydu** [comment]
Cám ơn @duythanhvn  đã góp ý nhé.
